src/docs/framework-integration/angular.md

```diff
@@ -89,193 +149,292 @@ import { angularOutputTarget } from '@stencil/angular-output-target';
 ```
 
 The name of your Stencil package should match the `name` property from the Stencil component library's `package.json`.
 
-Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the changes to the Angular component library.
+Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the
+changes to the Angular component library.
 
-> **NOTE**: As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy, however, will modify your `package.json` so it is important to make sure you do not commit those changes.
+:::note
+As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy,
+however, will modify your `package.json` so it is important to make sure you do not commit those changes.
+:::
 
 ## Consumer Usage
 
+### Creating a Consumer Angular App
+
+:::note
+If you already have an Angular app, skip this section.
+:::
+
+From your Angular workspace (`/packages/angular-workspace`), run the following command to generate an Angular application:
+
+```bash
+npx -p @angular/cli ng generate app my-app
+```
+
+### Consuming the Angular Wrapper Components
+
 This section covers how developers consuming your Angular component wrappers will use your package and component wrappers.
 
-If you distributed your components through a primary `NgModule`, developers can simply import that module into their implementation to use your components.
+In order to use the generated component wrappers in the Angular app, you'll first need to build your Angular component library. From the root
+of your Angular workspace (`/packages/angular-workspace`), run the following command:
+
+```bash
+npx -p @angular/cli ng build component-library
+```
+
+Now you can reference your component library as a standard import. If you distributed your components through a primary `NgModule`, you can
+simply import that module into their implementation to use your components.
 
 ```ts
-import { ExampleLibraryModule } from 'your-angular-library-package-name';
+// app.module.ts
+
+import { ComponentLibraryModule } from 'component-library';
 
 @NgModule({
-  imports: [ExampleLibraryModule],
+  imports: [ComponentLibraryModule],
 })
-export class FeatureModule {}
+export class AppModule {}
 ```
 
-Alternatively, developers can individually import the components and declare them on a module:
+Alternatively, you can individually import the components and declare them on a module:
 
 ```ts
-import { MyComponent } from 'your-angular-library-package-name';
+// app.module.ts
+
+import { MyComponent } from 'component-library';
 
 @NgModule({
   declarations: [MyComponent],
-  exports: [MyComponent],
 })
-export class FeatureModule {}
+export class AppModule {}
 ```
```

Comment:
Is this example complete? I run into build errors with:
```tsx
import { NgModule } from '@angular/core';

import { AppComponent } from './app.component';
import { MyComponent } from 'component-library';
@NgModule({
  declarations: [
    AppComponent,
    MyComponent,
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```

> Error: projects/my-app/src/app/app.component.html:427:3 - error NG8001: 'my-component' is not a known element:
> 1. If 'my-component' is an Angular component, then verify that it is part of this module.
> 2. If 'my-component' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.

Reply:
@rwaskiewicz Thinking about this a little more, I think we should remove this section from the docs. This approach has some additional configuration/setup needed to make sure elements get registered correctly because they wouldn't have a mechanism in place to leverage the loader for registering/loading the custom elements. 

Pulling in the whole component library module is the approach we spend more time hashing out. Alternatively, this approach would work better when using the `dist-custom-elements` output. So, we could move some of this to the section in the docs here where we talk about that.